build-support/flake8/.flake8

```diff
@@ -1,12 +1,26 @@
 [flake8]
 extend-ignore:
-  E203,  # whitespace before ':' (conflicts with Black)
-  E231,  # Bad trailing comma (conflicts with Black)
-  E501,  # line too long (> 79 characters)
-  E731,  # Do not assign a lambda expression
-  E741,  # Ambiguous variable name (enable once fixed)
-  W503,  # line break before binary operator  (conflicts with Black)
-  PB10,  # Bad class attribute (enable once fixed)
-  PB13,  # `open()` not within a context manager (possibly enable once fixed)
-  NIC002, # Implicitly concatenated string literals over multiple lines
-  NIC102, # Implicitly concatenated bytes literals over multiple lines
+  # whitespace before ':' (conflicts with Black)
```

Comment:
flake8 parses these per-line (not intelligently) so we were ignoring `E203, #, whitespace, ...`